source/localizable/smart_cart/_order_object.html.md.erb

```diff
@@ -38,3 +40,57 @@ Name                              | Type   | Value | Description
 `line_items[_].size.label`        | String | `"Νούμερο"` / `"Μέγεθος"` / `"Ηλικία"` |Item size label
 `line_items[_].size.value`        | String |    |Item size value
 `line_items[_].size.shop_value`   | String |    |Item original size as provided by the shop
+
+#### Accept options
+
+These are valid values to POST to the corresponding keys when
+[accepting a single order](/smart_cart/orders_api/#accept-a-single-order)
+(only available for orders with state `"open"`).
+
+Name                              | Type   | Example     | Description
+:-------------------------------: | :----: | :---------: | :------------------------------------------------:
+`accept_options.number_of_parcels`| Array  | `[1, 2, 3]` | Valid values to post on `number_of_parcels`
+`accept_options.pickup_location`  | Array  |             | [Pickup location](#accept-option-pickup-location)
+`accept_options.pickup_window`    | Array  |             | [Pickup window](#accept-option-pickup-window)
+
+#### Accept option: Pickup location
+
+Valid pickup locations to post when
+[accepting a single order](/smart_cart/orders_api/#accept-a-single-order) (use `id`).
+
+Name                    | Type    | Description
+:---------------------: | :-----: | :-----------------------:
+`pickup_location.id`    | Integer | The location ID to use
```

Comment:
[super minor] How about adding `accept_options[].` / `reject_options[].` before each attribute name for pickup locations, pickup windows and rejection reasons? It's not that important, just to be somewhat consistent with the rest of the objects descriptions (see line item sizes). Feel free to ignore :)

Reply:
These are not arrays therefore, that's why I skipped the `[]` part... I think I'll leave them like this for now, wdyt?